src/v2/stack_aggregator.py

```diff
@@ -361,3 +369,181 @@ def execute(request: Dict, persist=True):
         return {'aggregation': 'success',
                 'external_request_id': output.external_request_id,
                 'result': output_dict}
+
+
+class GoAggregator(Aggregator):
```

Comment:
Please move GoAggregator into separate module/file.

Reply:
As discussed, we are hitting cyclic import Error on this, and resolving it will increase the scope to a large extend. 
Probably, if needed we will make this change in the next iteration.